src/witan/models/dem/ccm/models.clj

```diff
@@ -72,7 +73,10 @@
      :witan/version "1.0.0"
      :witan/type :function
      :witan/fn :ccm-core/add-births}
-    {:witan/name :age-on, :witan/version "1.0.0", :witan/type :function, :witan/fn :ccm-cor/age-on}
+    {:witan/name :age-on
+     :witan/version "1.0.0"
+     :witan/type :function
+     :witan/fn :ccm-cor/age-on}
```

Comment:
I notice there's a missing `e` in `core` - please check the contract and fix in either location, than ks.